src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -88,6 +90,10 @@ public function add(array $parameters = array())
      */
     public function get($path, $default = null, $deep = false)
```

Comment:
IMO $path should be renamed to $key. It's not a path anymore when deep cannot/should not be used.